espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/drivers/AppDriver.kt

```diff
@@ -0,0 +1,12 @@
+package io.appium.espressoserver.lib.drivers
```

Comment:
license header is missing. Please check it for all newly added files

Reply:
added to all new files